app/client/components/identity/useConsentOptions.ts

```diff
@@ -78,7 +86,7 @@ const reducer = (state: State, action: Action) => {
 };
 
 export const Actions = {
-  error: () => ({ type: ActionType.ERROR }),
+  error: (payload: Error) => ({ type: ActionType.ERROR, payload }),
```

Comment:
if you wanted to use this approach for non-exception error messages the type here could be `string | Error` and `Raven.captureException()` would handle it. Not needed yet though.